### DISCUSSION TITLE
Elementor not working in vercel wordpress

### DISCUSSION THREAD

**Discussion by belbaz** [mitchmac/ServerlessWP#48] [General]

**Comment by mitchmac** [comment]
Hi @belbaz -- let's try to figure out what's not working!

When you say that elementor plugins are in the WordPress plugins folder, do you mean that you added them to the Git repository for your site?

There may be helpful error messages in two spots. In your web browser's developer tools there may be error messages in the Console. Also in your Vercel project for the site, under "Logs" there may be something suspicious noted. That's where I would check first.
